libafl_targets/src/coverage.c

```diff
@@ -9,8 +9,22 @@ typedef uint32_t prev_loc_t;
 #define CTX_MAX_K 32U
 
 extern uint8_t __afl_area_ptr_local[EDGES_MAP_SIZE];
+
 uint8_t* __afl_area_ptr = __afl_area_ptr_local;
 
+
+// Weak symbols, LLVM Passes overwrites them if we really use it
+#ifdef __linux__
+extern uint8_t __attribute__((weak)) __start_libafl_token;
```

Comment:
In common.h we handle weak symbols for functions https://github.com/AFLplusplus/LibAFL/blob/main/libafl_targets/src/common.h#L102 and I guess you can adapt the code for globals

Reply:
ok I'll check it out